code/modules/antagonists/nukeop/equipment/borgchameleon.dm

```diff
@@ -50,7 +50,7 @@
 		else
 			to_chat(user, span_warning("You can't use [src] while inside something!"))
 	else
-		to_chat(user, span_warning("You need at least [activationCost] charge in your cell to use [src]!"))
+		to_chat(user, span_warning("You need at least [activationCost / (1 KILO JOULES)]KJ charge in your cell to use [src]!"))
```

Comment:
```suggestion
		to_chat(user, span_warning("You need at least [display_energy(activationCost)] of charge in your cell to use [src]!"))
```

You shouldn't scale this. Joule = joule. The old description lied because they assumed the cell used joules I guess.

Also display_energy() automatically chooses the suitable prefix, so you can use that here.